### PR TITLE
First exercise

### PR DESCRIPTION
**Pull request by valentino-cachan**
Screenshot

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://cdn.tailwindcss.com?plugins=forms,typography"></script>
```
Comment (facundo-cachan) [inline index.html]: Remove JS

File: index.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+		<script src="https://unpkg.com/unlazy@0.11.3/dist/unlazy.with-hashing.iife.js" defer init></script>
+		<script type="text/javascript">
```
Comment (facundo-cachan) [inline index.html]: Remove tailwind

File: index.html [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+							card: {
+								DEFAULT: 'hsl(var(--card))',
+								foreground: 'hsl(var(--card-foreground))'
+							},
+						},
+					}
+				}
+			}
+		</script>
+		<!-- <style type="text/tailwindcss">
```
Comment (facundo-cachan) [inline index.html]: Remove unused code

**Review by facundo-cachan — CHANGES_REQUESTED** [review]

**Comment by facundo-cachan** [comment]
Format code

File: index.html (hunk 3 above)
Comment (valentino-cachan) [reply index.html]: fixed

**Review by facundo-cachan — APPROVED** [review]

**Pull request merged**